internal/interface/web/middleware.go

```diff
@@ -0,0 +1,26 @@
+package web
+
+import (
+	"time"
+
+	sentrygin "github.com/getsentry/sentry-go/gin"
+	"github.com/gin-gonic/gin"
+	log "github.com/sirupsen/logrus"
+)
+
+// SetupMiddleware configures and adds the Sentry middleware to the Gin engine
+// This middleware handles error reporting and panic recovery in a single package
+func SetupMiddleware(engine *gin.Engine, enabled bool) {
```

Comment:
doesn't need to be an exported function
```suggestion
func setupMiddleware(engine *gin.Engine, enabled bool) {
```

Reply:
Done